apps/heft/src/plugins/TypeScriptPlugin/TypeScriptBuilder.ts

```diff
@@ -274,13 +265,10 @@ export class TypeScriptBuilder extends SubprocessRunnerBase<ITypeScriptBuilderCo
         throw new Error('Unable to resolve "tslint" package');
       }
 
-      const tslintScopedLogger: IScopedLogger = await this.requestScopedLoggerAsync(
-        loggerPrefixLabel ? `tslint (${loggerPrefixLabel})` : 'tslint'
-      );
+      const tslintScopedLogger: IScopedLogger = await this.requestScopedLoggerAsync('tslint');
```

Comment:
Parallel to `typescriptLogger` above:
```suggestion
      const tslintLogger: IScopedLogger = await this.requestScopedLoggerAsync('tslint');
```